src/main/java/qna/domain/Answer.java

```diff
@@ -1,17 +1,34 @@
 package qna.domain;
 
+import org.springframework.data.annotation.LastModifiedDate;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+public class Answer extends BaseEntity{
```

Comment:
```suggestion
public class Answer extends BaseEntity {
```
다른 클래스의 포맷을 보니, 요기는 띄어쓰기가 깨진 것 같아요~
포맷팅 자동화를 하지 않으신다면, PR전에 눈으로라도 한번 확인하는 것도 좋습니다~
팀원들과 같이 작업하신 거라면, 눈이 여러개인만큼 혼자 작업할때보다 더 잘 찾으실 수 있을 거에요!😆